src/Microsoft.Content.Build.MerfToSDPForJava.Interpreter/MethodExpression.cs

```diff
@@ -27,7 +27,7 @@ public override bool Interpret(PageModel pageModel, BuildContext context)
                 return true;
 
             var methodsOverLoading = methods.Where(item => item != null && item.Overload != null && item.Overload.EndsWith('*'))?.ToList();
-            if (methodsOverLoading != null)
+            if (methodsOverLoading == null)
```

Comment:
what is expect behavior here?

Reply:
if methodsOverLoading == null   {return true} , Otherwise, the subsequent code will not be executed at all